test/Sentry.Hangfire.Tests/Sentry.Hangfire.Tests.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net6.0</TargetFrameworks>
```

Comment:
Not testing the .NET Framework build?

Reply:
I forgot to add the explainer: Hangfire is not strong naming their assemblies and CI chokes on Windows on net48. Due to the viral nature and the use of `InternalsVisibleTo` I could not get it to opt out.